Userland/Applications/Escalator/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+serenity_component(
+    Escalator
+    REQUIRED
+    TARGETS Escalator
+)
+
+compile_gml(Escalator.gml EscalatorGML.h escalator_gml)
+
+set(SOURCES
+    main.cpp
+    EscalatorWindow.cpp
+    EscalatorGML.h
+)
```

aha! a nitpick.

CxByte pushed a [commit](https://github.com/SerenityOS/serenity/commit/166a905951023a4c183a196ef444e689fdfa9697) yesterday moving all generated headers in SOURCES into `GENERATED_SOURCES` where they belong

```suggestion
set(SOURCES
    main.cpp
    EscalatorWindow.cpp
)

set(GENERATED_SOURCES
    EscalatorGML.h
)
```

Then I think we'll be done, I was waiting on Tim to hit the approve button but this is probably been over-reviewed at this point :)